src/tree/ContainerAppTreeItem.ts

```diff
@@ -31,6 +31,7 @@ export class ContainerAppTreeItem extends AzExtParentTreeItem implements IAzureR
 
     public name: string;
     public label: string;
+    public childTypeLabel: string = localize('containerAppSetting', 'Container App setting');
```

Comment:
What was the reason for this change?? I assume that it's for a quickpick thing, but not all of the children will be `Container App setting`

Reply:
Yup it's a quick pick related thing. I believe walking through the openLogsInPortal command via the command palette made me want to add this. I didn't want the QP prompt showing up as undefined for the label. Is there a better label you'd prefer to use?